### DISCUSSION TITLE
A crash caused by the dashboard not starting

### DISCUSSION THREAD

**Discussion by skyDuanXianBing** [microsoft/aspire#3704] [Q&A] [answered]
I used aspire to work normally before, after a while, today when I rebooted, the dashboard wouldn't start, causing my program to crash
![屏幕截图 2024-04-15 161022](https://github.com/dotnet/aspire/assets/164180639/009a6a2e-995a-4bd3-80f4-19a06e69755f)

**Comment by skyDuanXianBing** [comment]
![image](https://github.com/dotnet/aspire/assets/164180639/98f7ed01-39ca-4a05-856e-fb628cd8f1e4)
Why is projects unavailable after I updated to.NET apphost

**Comment by skyDuanXianBing** [comment]
I fix this problem, to copy new aspires' project properties launchsetting to my old aspires' project

**Comment by davidfowl** [comment, marked as answer]
You're running preview 4, please update to preview 5. Here are the update instructions https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-5?tabs=dotnet-cli#upgrade-to-preview-5
